docs/integrations/lightgbm.rst

```diff
@@ -1,80 +1,105 @@
 Neptune-LightGBM Integration
 ============================
 
-This integration lets you customize training scripts written in |LightGBM| to log metrics to Neptune.
-
+|lightGBM| is a popular gradient boosting library. The integration with Neptune lets you log training and evaluation metrics and have them visualized in Neptune.
 
 .. image:: ../_static/images/others/lightgbm_neptuneml.png
    :target: ../_static/images/others/lightgbm_neptuneml.png
    :alt: lightGBM neptune.ai integration
 
-Say your training script looks like this:
+Requirements
+------------
+To use Neptune + lightGBM integration you need to have installed is |neptune-client| and |neptune-contrib|.
+
+.. code-block:: bash
+
+    pip install neptune-client neptune-contrib['monitoring']
```

Comment:
Are you sure those quotes are necessary?

Reply:
Nope :) I dropped them.